posthog/tasks/usage_report.py

```diff
@@ -684,7 +684,7 @@ def get_teams_with_api_queries_metrics(
     query = f"""
         SELECT JSONExtractInt(log_comment, 'team_id') team_id, count(1) cnt, sum(read_bytes) read_bytes
         FROM clusterAllReplicas({CLICKHOUSE_CLUSTER}, system.query_log)
```

Comment:
Hopefully using the unreplicated `query_log` table here is not the long-term plan